### ISSUE TITLE
Mouser BOM misses R1-R4 (10k 1206 resistors)

### ISSUE THREAD

**Issue by fd0** [kinx-project/mk66f-hw#1]
Hey, I just noticed that the BOM linked for Mouser misses the resistors R1-R4, 10k 1206. Is that intentional? :)

**Comment by stapelberg** [comment]
Thanks for reporting this! I’ll fix it later today. No intention here :)

**Comment by fd0** [comment]
btw, here's a [CSV](https://gist.github.com/ab9dd47c5fb4b62c508c8dad4bdded72) with most of the parts for Digikey, I'm not sure how useful that is for other people though. Some parts I had already orderer from elsewhere, most part numbers from the Mouser list can directly be used to find the parts on Digikey.

**Comment by stapelberg** [comment]
Just got a chance to take a look at this.

https://github.com/kinx-project/mk66f-hw/blob/9843ece854f8d0d39c7284b8e975a7eaf654a99e/bom.mouser.csv#L17 are the resistors, see https://www.mouser.ch/ProductDetail/Yageo/RC1206FR-0710KL?qs=%252BhB4WxZmbVEJV82xdVgRUw==

Did you miss that line, or was there an issue with importing the CSV file somewhere?

**Comment by stapelberg** [comment]
> btw, here's a [CSV](https://gist.github.com/ab9dd47c5fb4b62c508c8dad4bdded72) with most of the parts for Digikey, I'm not sure how useful that is for other people though. Some parts I had already orderer from elsewhere, most part numbers from the Mouser list can directly be used to find the parts on Digikey.

Just to be 100% clear: is that BOM complete (you mentioned you have ordered some parts elsewhere)? If yes, could you send it in a PR please? It’s certainly nice to have a BOM for digikey :)

**Comment by fd0** [comment]
> Did you miss that line, or was there an issue with importing the CSV file somewhere?

Ah, maybe my description was misleading (sorry for that): the resistors are missing from the linked project basket at Mouser, here: https://www.mouser.ch/ProjectManager/ProjectDetail.aspx?AccessID=7798be8d00 

That's the one I used for placing an order at Digikey.

> Just to be 100% clear: is that BOM complete (you mentioned you have ordered some parts elsewhere)?

I've ordered the connectors for the Kinesis PCBs elsewhere, so they aren't included. I'll revisit this issue once the parts arrive, then I know if something else is missing, if I find some time I'll create a PR.
